website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -16,6 +16,62 @@ upgrade flow.
 
 ## Consul 1.11.0
 
+### Deprecated Agent Config Options
+
+Consul 1.11.0 is compiled with Go 1.17 and now the ordering of
+`tls_cipher_suites` will no longer be honored. Additionally
+`tls_prefer_server_cipher_suites` is now ignored.
+
+The `master` and `agent_master` ACL tokens in the `acl.tokens` config block
+have been renamed to `initial_management` and `agent_recovery` respectively.
```

Comment:
We need to updated https://www.consul.io/docs/security/acl/acl-system#acl-tokens and remove `master` and `agent_master`

Reply:
We're not on versioned docs yet, so when we do that we should keep that in mind for anyone on older versions trying to use that page.

Reply:
We need at least call it out as renamed in 1.11.0 as we have done in other places @rboyer

Reply:
That was already handled on `main`: https://github.com/hashicorp/consul/blob/main/website/content/docs/security/acl/acl-system.mdx

Reply:
@karl-cardenas-coding we're going to force push `main` onto `stable-website` after the release to catch that up